### DISCUSSION TITLE
Looking forward to “Navigation rail”

### DISCUSSION THREAD

**Discussion by d3vw** [material-components/material-web#5159] [General]
https://m3.material.io/components/navigation-rail/overview

When will it come?

**Comment by datvm** [comment]
Not in the near future I assume: [Roadmap](https://github.com/material-components/material-web/blob/main/docs/roadmap.md#future)

**Comment by 0xENDER** [comment]
Seems like it may take another year if things keep progressing at the same pace..

**Comment by treeder** [comment]
We're working on it here: https://github.com/treeder/material/issues/3
